Resources/Prototypes/_Gabystation/jukebox.yml

```
@@ -190,3 +190,20 @@
   path:
     path: /Audio/_Gabystation/Jukebox/winds_of_fjord.ogg
 
+- type: jukebox
+  id: Cumade e Cumpade
+  name: Leandro E Leonardo - Cumade e Cumpade 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)